specification/glossary.md

```diff
@@ -2,10 +2,130 @@
+
+The total amount of fees and discounts as documented in an itemized bill.
+
+## Billing Account
+
+A mandatory account established with a service provider to manage and pay for all charges associated with an invoice. A billing account might have sub-accounts, all of whose costs are consolidated to the billing account.
+
+## Billing Currency
+
+The specific currency in which invoices or bills for services are issued and payments are expected to be made.
+
+## Billing Period
+
+The time interval over which a customer's usage of services is measured for billing purposes. For most cloud providers, the billing period is a month.
+
+## Block Pricing
+
+Pricing approach where the cost of a particular resource or service is determined based on predefined quantities or tiers of usage. In these scenarios, the Pricing Quantity can be different from the Usage Quantity.
+
+## Charge
+
+A row in a FOCUS compatible cost and usage data associated with the usage of a service.
+
+## Commitment
+
+A customer's agreement to consume a specific quantity of a service or resource over a defined period, usually also creating a financial commitment throughout the entirety of the commitment period. Some commitments also hold Providers to certain assurance levels of resource availability.
+
+## Commitment-Based Discount
+
+Also known as Commitment Discount, this is a pricing strategy or incentive offered by a service provider to encourage customers to commit to a certain level of usage or purchase over a specified period. Offerings vary in specificity of commitment, penalties for underutilization, and possible remedies.
+
+## CSP
+
+Cloud Service Provider (e.g. Amazon Web Services, Microsoft Azure, Google Cloud Platform).
+
+## Dimension
+
+Specification-defined categorical attribute that provides context or categorization to billing data.
+
+## Effective Cost
+
+A comprehensive and realistic measure of the true cost of a service, inclusive of all reduced rates, discounts, and impacts of amortization.
+
+## Finalized Tag
+
+A tag with one tag value chosen from a set of possible tag values after being processed by a set of provider-defined or user-defined rules.
+
+## FOCUS
+
+FinOps Cost and Usage Specification - An effort to build an open specification for cloud billing data.
+
+## Interruptible
+
+Category of compute resources that can be paused or terminated by the CSP within certain criteria, often available for discounted pricing when compared to the equivalent non-interruptible resource.
+
+## List Unit Price
+
+Provider published unit price for a single pricing measurement unit.
+
+## Metric
+
+A quantifiable, standardized measure used to assess, evaluate, or track the performance, quality, or characteristics of a system, process, or service.
+
+## MSP
+
+Managed Service Provider - A company or organization that provides outsourced management and support of a range of IT services, such as network infrastructure, cybersecurity, cloud computing, and more.
```

Comment:
Should we include "Managed Service Provider" as part of the term?

MSP (Managed Service Provider)